src/resources/postcss/components/_events-bar.pcss

```diff
@@ -159,6 +159,40 @@
 		width: 21px;
 	}
 
+	.tribe-events-c-events-bar__tabs {
+		display: flex;
+	}
+
+	.tribe-events-c-events-bar__tab {
```

Comment:
there should be a 8px margin between the two tabs (it's a bit hard to see, but if you focus on the text you'll get the width of the tabs)